kotlin-reference-server/src/main/kotlin/org/stellar/reference/sep24/Sep24Route.kt

```diff
@@ -133,16 +134,16 @@ fun Route.sep24(
           }
           else ->
             call.respond(
-              ErrorResponse("The only supported operations are \"deposit\" or \"withdrawal\""),
+              MessageResponse("The only supported operations are \"deposit\" or \"withdrawal\""),
             )
         }
       } catch (e: ClientException) {
         log.error(e)
-        call.respond(ErrorResponse(e.message!!))
+        call.respond(MessageResponse(e.message!!))
```

Comment:
Why ErrorRespond was changed to MessageRespond in places where exceptions are caught?

Reply:
It was renamed from `ErrorResponse` to `MessageResponse` just because it has field called `msg` and i wanted to add new error response with `error` field. But now it looks like it is redundant, so i will use only old ErrorResponse